client/my-sites/domains/domain-management/transfer/transfer-to-other-site/index.jsx

```diff
@@ -115,52 +113,59 @@ export class TransferToOtherSite extends React.Component {
 		}
 	};
 
-	render() {
-		if ( ! this.isDataReady() ) {
-			return <DomainMainPlaceholder goBack={ this.goToEdit } />;
+	getMessage() {
+		const { selectedDomainName: domainName, isMapping, translate } = this.props;
+		const translateArgs = { args: { domainName }, components: { strong: <strong /> } };
+
+		if ( isMapping ) {
+			return translate(
+				"Please choose a site you're an administrator on to transfer domain mapping for {{strong}}%(domainName)s{{/strong}} to:",
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Please choose a site you're an administrator on to transfer {{strong}}%(domainName)s{{/strong}} to:' )` *ES Score: 6*